docs/shared/devkit-and-nx-plugins.md

```diff
@@ -27,7 +27,7 @@ Plugins have:
 
   - Plugins can provide a function `processProjectGraph` to add extra edges to the project graph.
   - This allows plugins to influence the behavior of `nx affected` and the project graph visualization.
-  - See [project graph plugins](/structure/project-graph-plugins) for more information.
```

Comment:
I see the `/structure/project-graph-plugins` is still working is it intended? Will this page remain with the addition of `extending-nx/project-graph-plugins`?

_I ask the question because I need to know if we need to add a redirection rule or not._

Reply:
I think we should add a redirect and have only one url for the project-graph-plugins page

Reply:
Moved this portion of the effort into #9758